gibberish_server/controllers/room_controller.js

```diff
@@ -4,14 +4,13 @@ const roomModel = require("../models/Room")
 
 class RoomController {
     static getRoomData(req, res) {
-        let r = roomModel.getRoomInfoAsObject(req.params["roomId"]);
-        r.nextState();
-        roomModel.saveRoom(r);
-        return roomModel.getRoomInfoAsJson(req.params["roomId"])
```

Comment:
the api will return json, not room object

Reply:
the code has been deleted